strapi/strapi
# prevent caption and alternativeText from automatically populating with file name on upload

## Fix for #13295 

### What does it do?

This is a reference to #13295 which confirms that the `caption` and `alternativeText` fields should not be populated with the file name on upload.

### Why is it needed?

The `caption` and `alternativeText` fields should not be populated with the file name on upload because these are not good practices for frontend web accessibility and user experience.

### How to test it?

After confirming a file for upload in the media upload interface, the interface automatically populates the `caption` and `alternativeText` fields with the file name. This PR prevents populating those fields with the file name and those fields should be blank after confirming a file to upload.

### Related issue(s)/PR(s)

#13295
